frontend/src/console/components/CustomPages/CustomPages.tsx

```diff
@@ -0,0 +1,202 @@
+    if (
+      prevProps.activeInstance.data == null &&
+      this.props.activeInstance.data
+    ) {
+      // eslint-disable-next-line
+      this.setState({
+        pageContent: this.getPageContentFromState()
+      });
+    }
+  };
+
+  getApiPageName = () => {
+    const pageName = capitalizeFirstLetter(this.props.match.params.pageName);
+    return `staticTemplate${pageName}Content` as keyof StaticContentOverrides;
+  };
+
+  getPageContentFromState = () => {
+    let content = '';
+    const instance = this.props.activeInstance;
+
+    if (
+      instance &&
+      instance.data &&
+      instance.data.draftStaticContentOverrides
+    ) {
+      const statePageContent =
+        instance.data.draftStaticContentOverrides[this.getApiPageName()];
+      if (typeof statePageContent === 'string') {
+        content = statePageContent;
+      }
+    }
+
+    return content;
+  };
+
+  hasContentChanged = () => {
+    return this.state.pageContent !== this.getPageContentFromState();
+  };
+
+  onChangeContent = (newContent: any) => {
+    this.setState({
+      pageContent: newContent
+    });
+  };
+
+  saveChanges = () => {
+    if (this.hasContentChanged()) {
+      this.props.updateStaticContentOverridesFieldValue(
+        this.props.activeInstance.data!.id,
+        this.getApiPageName(),
+        this.state.pageContent
+      );
+    }
+  };
+
+  public render() {
+    const instance = this.props.activeInstance;
+    const { pageName } = this.props.match.params;
+
+    // TODO: The hide/show switch is just a placeholder component
+    // We still need to add override capability to Ocim to disable
+    // custom pages and hide links.
+
+    return (
+      <ConsolePage contentLoading={this.props.loading}>
+        <ConsolePageCustomizationContainer>
+          <div className="custom-page-editor">
+            <Row>
+              <Col md={8}>
+                <h2>
+                  <WrappedMessage messages={messages} id={pageName} />
+                </h2>
+              </Col>
+              <Col className="page-switch-column">
+                {/* This is a placeholder switch! This should be enabled when we  */}
+                {/* implement the link overriding and page hiding features.  */}
+                <Form className="page-switch-form">
+                  <Form.Label htmlFor="show-page-switch">
+                    <WrappedMessage messages={messages} id="showPage" />
+                  </Form.Label>
+                  <Form.Check
+                    disabled
+                    checked
+                    type="switch"
+                    label=""
+                    id="show-page-switch"
+                  />
+                </Form>
+              </Col>
+            </Row>
+
+            {/* Uncomment when the backend properly passes back LMS URL */}
+            {/* <a className="pageLink" href="http://google.com">
+              http://google.com
+            </a> */}
```

Comment:
Nit: could we leave a placeholder link here (e.g. just a link to opencraft.com) instead of hiding it, like we did with the toggle switch?

Reply:
I'm adding the LMS and studio endpoints to the API. :)